db/python/layers/family.py

```diff
@@ -162,6 +191,7 @@ async def get_pedigree(
         # pylint: disable=invalid-name
         replace_with_family_external_ids=False,
         empty_participant_value='',
+        include_header=True,
```

Comment:
Should this default to True here?  It looks like it's set as `False` by default here https://github.com/populationgenomics/sample-metadata/blob/d3b75a5cec9f54e606403d2ea9a7a66c5c15f196/api/routes/family.py#L51. 
That appears to be the only place that `get_pedigree` is called.

Reply:
I'm not too sure of the use case where `include_header` will be set to True as default as it seems it will always be provided when called?

Reply:
Nice catch, I've actually flipped this around. I think it makes sense for the route to default for include_header to True, but if you're requesting the pedigree around sample-metadata, you probably don't want the header.